python/lsst/ts/tunablelaser/canbus_modules.py

```diff
@@ -45,13 +45,39 @@
     "MidiOPG",
     "E5DCB",
 ]
+# TODO: (DM-46168) Revert workaround for TunableLaser XML changes
+import enum
 import logging
+import warnings
 
 from . import interfaces
 from .compoway_register import CompoWayFDataRegister, CompoWayFOperationRegister
-from .enums import Mode, OpticalConfiguration, Output, Power
+from .enums import Mode, Output, Power
 from .register import AsciiRegister
 
+try:
```

Comment:
you are doing this in multiple places, defining the same enumeration, and you are then removing the enum from the enums.py module. 

Consider leaving this alone and just adding a "TODO" in the enums.py module to replace the enumeration with the one from ts-xml once that becomes available.